### PR TITLE
docs: add basic docs site

### PR DESCRIPTION
**Pull request by luwes**
This adds a basic Astro docs site. 
Test link: https://media-chrome-docs-git-fork-luwes-docs-site-mux.vercel.app/en/get-started

### TODO

- [x] Add Get Started page content
- [ ] Get right side bar table of contents header highlight working
- [ ] Try to remove React for just the Algolia searchbox, there is a vanilla JS component available as well.

### CODE REVIEW

**Comment by heff** [comment]
Awesome! Love it.

I think it's good the docs are in the same repo, because it's gonna take extra attention to keep them up to date now.

In the element docs we're linking out to the examples pages, but could we ultimately just build them into the docs page? (not a request for the first pass)

File: docs/src/components/HeadCommon.astro [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<!-- Preload Fonts -->
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+  href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,700;1,400&display=swap"
+  rel="stylesheet"
+/>
+
+<!-- Scrollable a11y code helper -->
+<script src="/make-scrollable-code-focusable.js" is:inline></script>
```
Comment (gkatsev) [inline docs/src/components/HeadCommon.astro:26]: not-blocking: this seems to be running too early, so, pre elements aren't getting `tabindex` applied to them.
Tested on https://media-chrome-docs-git-fork-luwes-docs-site-mux.vercel.app/en/media-controller#styling

Comment (gkatsev) [reply docs/src/components/HeadCommon.astro:26]:
```suggestion
<script src="/make-scrollable-code-focusable.js" is:inline defer></script>
```

Deferring the script seems to fix it.

File: docs/src/config.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+export const SITE = {
+  title: 'Media Chrome Docs',
+  description: 'Your website description.',
```
Comment (gkatsev) [inline docs/src/config.ts:3]:
```suggestion
  description: 'Documentation for Media Chrome, a set of fully customizable media player controls using web components.',
```

File: docs/src/config.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+export const GITHUB_EDIT_URL = `https://github.com/muxinc/media-chrome/tree/main/docs`;
+
+export const COMMUNITY_INVITE_URL = `https://github.com/muxinc/media-chrome/discussions`;
+
+// See "Algolia" section of the README for more information.
+export const ALGOLIA = {
+  indexName: 'dev_media_chrome',
+  appId: '57LJZR6XKN',
+  apiKey: 'ec6de967b91bcd3b0ea96b5fcfedaa7d',
+};
```
Comment (gkatsev) [inline docs/src/config.ts:40]: Is this hooked up? Not getting any results. Maybe it needs to be more officially deployed?

File: docs/src/pages/en/media-controller.md [hunk 4]
```diff
@@ -319,26 +323,28 @@ As we work through other common use cases, both internally and with the communit
...
-| `m`     | Toggle mute       |
-| `f`     | Toggle fullscreen |
-| ⬅       | Seek back 10s     |
-| ➡       | Seek forward 10s  |
+| Space | Toggle Playback |
+| `k` | Toggle Playback |
+| `m` | Toggle mute |
+| `f` | Toggle fullscreen |
+| ⬅ | Seek back 10s |
+| ➡ | Seek forward 10s |
```
Comment (gkatsev) [inline docs/src/pages/en/media-controller.md:337]:
```suggestion
| ⬅️ | Seek back 10s |
| ➡️ | Seek forward 10s |
```

File: docs/src/pages/en/media-volume-range.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (gkatsev) [inline docs/src/pages/en/media-volume-range.md:1]: no-action: weird that github didn't detect this page as moved

File: docs/src/pages/en/styling.md [hunk 6]
```diff
@@ -10,14 +14,14 @@ Our current styling architecture is still quite nascent and is very likely to un
...
 
-- `<media-captions-button/>` ([docs](./media-captions-button.md))
-- `<media-fullscreen-button/>` ([docs](./media-fullscreen-button.md))
-- `<media-mute-button/>` ([docs](./media-mute-button.md))
-- `<media-pip-button/>` ([docs](./media-pip-button.md))
-- `<media-play-button/>` ([docs](./media-play-button.md))
-- `<media-playback-rate-button/>` ([docs](./media-playback-rate-button.md)) ([See notes below \*](#notes))
-- `<media-seek-backward-button/>` ([docs](./media-seek-backward-button.md))
-- `<media-seek-forward-button/>` ([docs](./media-seek-forward-button.md))
+- `<media-captions-button/>` ([docs](./media-captions-button))
```
Comment (gkatsev) [inline docs/src/pages/en/styling.md:17]: too bad that we can have this link continue working in github too. If you figure it out, that'll be nice, but I wouldn't spend more than like a minute on this.

File: docs/src/pages/index.astro [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<script is:inline>
+  // Redirect your homepage to the first page of documentation.
+  // If you have a landing page, remove this script and add it here!
+  window.location.pathname = `/en/get-started`;
```
Comment (gkatsev) [inline docs/src/pages/index.astro:4]: I guess some version of media-chrome.org like thing would eventually replace this?

**Review by gkatsev — APPROVED** [review]
This is great! Had a couple of tiny things I noticed, but not blockers and can always be done as follow-up PRs.

File: docs/media-volume-range.md [hunk 8]
```diff
@@ -1,18 +0,0 @@
-# `<media-volume-range/>`
```
Comment (cjpillsbury) [inline docs/media-volume-range.md:1]: question(non-blocking): Any idea why did this not get tracked as a move?

**Comment by cjpillsbury** [comment]
Since there's a bit of a 🐔 & 🥚 problem for this, let's plan on a fast follow that updates our readme.md once we have a stable URL for the docs deploy so we can 🐣.

File: docs/src/pages/en/media-volume-range.md (hunk 5 above)
Comment (cjpillsbury) [reply docs/src/pages/en/media-volume-range.md:1]: heh just saw you commented on this on the other side of the fence.

**Review by cjpillsbury — APPROVED** [review]
😍

File: docs/media-volume-range.md (hunk 8 above)
Comment (luwes) [reply docs/media-volume-range.md:1]: hmm not sure, will check this out

File: docs/src/pages/en/styling.md (hunk 6 above)
Comment (luwes) [reply docs/src/pages/en/styling.md:17]: ah yes, that would be nicer. I'll check this out

**Pull request merged**